wallet/how-to/use-non-evm-networks/starknet/send-starknet-transactions.md

```diff
@@ -0,0 +1,209 @@
+---
+description: Send Starknet transactions in MetaMask.
+sidebar_position: 4
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Send Starknet transactions
+
+You can send Starknet transactions using the
+[`get-starknet`](https://github.com/starknet-io/get-starknet) library or the
+[`wallet_invokeSnap`](/snaps/reference/wallet-api-for-snaps/#wallet_invokesnap) JSON-RPC method.
+
+## Prerequisites
+
+[Connect to Starknet](connect-to-starknet.md) from your dapp.
+
+## Send a transaction
+
+Send a transaction using the
+[`starknet.account.execute()`](https://starknetjs.com/docs/api/classes/accountinterface/#execute)
+function (with `get-starknet`) or the
+[`starknet_executeTxn`](../../../reference/non-evm-apis/starknet-snap-api.md#starknet_executetxn)
+method (with `wallet_invokeSnap`):
+
+<Tabs>
+  <TabItem value="get-starknet" default>
+
+  ```javascript
+  const sendStarknetTransaction = async (wallet, contractAddress, entrypoint, calldata) => {
+    try {
+      if(wallet?.isConnected !== true){
+        throw("Wallet not connected");
+      } 
+    
+      // Send the transaction.
+      const result = await wallet?.account?.execute({
+        contractAddress: contractAddress,  // The address of the contract.
+        entrypoint: entrypoint,            // The function to call in the contract.
+        calldata: calldata                 // The parameters to pass to the function.
+      });
+      console.log("Transaction successful:", result);
+      return result;
+    } catch (error) {
+      console.error("Error sending transaction:", error);
+    }
+  };
+  ```
+
+  </TabItem>
+  <TabItem value="wallet_invokeSnap">
+
+  ```javascript
+  const sendStarknetTransaction = async (contractAddress, contractFuncName, contractCallData, address, chainId, maxFee = null) => {
```

Comment:
Should these be updated to match the variables used later in this code snippet for the `executeTxn` params?

In general this code sample should be reviewed for syntax and accuracy.